WooCommerce.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,6 +1,15 @@
 {
-  "originHash" : "9ea1ab4271974e9606eddf01e2528d962e975d6abd3fb314fae28d3eeeb07fe9",
+  "originHash" : "30ecaceed4e1b487697506426e20d60843bbce1a647790dd9bc8b78d8b2edcf7",
```

Comment:
I commented in #15654 that I'm surprised we still have the `Package.resolved` in the workspace when there are no longer package dependencies there, only the internal Modules package.

Xcode keeps generating this file, so there must be something still configured for it in there.

I'm delaying tackling this inconsistency till after having moved everything to SwiftPM. It should be simpler to diagnose then, with less moving parts in Xcode itself.